package.json

```diff
@@ -57,7 +59,9 @@
   },
   "dependencies": {
     "@capacitor-community/bluetooth-le": "3.0.0",
+    "@capacitor/android": "5.0.4",
     "@capacitor/core": "5.0.4",
+    "@capacitor/ios": "5.0.4",
```

Comment:
can you bump to 5.6.0 ? would be cool to update core as well.

Reply:
No, I expect before this gets merged, there will be at least 10 new version. I know this version works with current version of https://github.com/oasisprotocol/ionic-ledger-hw-transport-ble. And currently in this PR there is no real way to test on real devices, as Bluetooth UI is still missing - I was planning to add it after this PR is merged.